src/java.xml/share/classes/module-info.java

```diff
@@ -229,16 +249,105 @@
  * </li>
  * </ul>
  *
+ * <h2 id="PP">Property Precedence</h2>
+ * Properties in this module can be set in multiple ways, including using
+ * the API properties, system properties, <a href="#ConfigurationFile">JAXP Configuration File</a>,
+ * the {@link javax.xml.XMLConstants#FEATURE_SECURE_PROCESSING FEATURE_SECURE_PROCESSING}
+ * (hereafter referred to FSP), and the default values. The order of precedence
+ * for the configuration sources is defined as follows, with earlier ones overriding the later:
+ *
+ * <ul>
+ * <li><p>
+ *      Properties specified through factories or processors API;
+ * </li>
+ * <li><p>
+ *      Properties set using the corresponding System properties;
```

Comment:
Can you confirm that this means system properties set on the command line with -Dkey=value or System.setProperty(key, value)?  

(I understand that the catalog feature "RESOLVE" property helps to explain it but I'm concerned that "system property" is ambiguous throughout).

Reply:
Yes. Took it as granted "system property" is well-understood. If it can still be confusing, added a section "Properties and System Properties" to make it clearer what "system property" meant throughout the document.